feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/cmd_crashlog.c

```diff
@@ -39,7 +39,6 @@ pid_t cmd_handler_crashlog(struct task *task)
 	_url = SCHEMA_KEY_VAL(task->conf.payload, "ul_url");
 	if (!_url) {
 		LOG(ERR, "curl: command without a valid ul_url");
```

Comment:
This could be a NOTICE instead of ERR.  The issue is with data in the command not AP operation.  The call TRACEBACK generated by the ERR log is noise and not helpful.  Also the message could specify that it is in the crash log command.

Reply:
yes, thank you